src/app/OperationalSessionSetup.h

```diff
@@ -189,8 +189,12 @@ class DLL_EXPORT OperationalSessionSetup : public SessionEstablishmentDelegate,
      * `onFailure` may be called before the Connect call returns, for error
      * cases that are detected synchronously (e.g. inability to start an address
      * lookup).
+     *
+     * `useTransportForLargePayload` is set to true when the session needs to be established
+     * over a transport that allows large payloads to be transferred, e.g., TCP.
```

Comment:
Would be better to have a flags bitmap or params struct or something so the naming is clearer at the callsite.  If a callsite is passing `true`, it's hard to tell what that means without reading this code.